luaui/Widgets/gui_top_bar.lua

```diff
@@ -2056,3 +2406,53 @@ function widget:SetConfigData(data)
 		autoHideButtons = data.autoHideButtons
 	end
 end
+
+
+function findBPCommand(unitID, ...) -- for bp bar only most likely
+    local commands = Spring.GetUnitCommands(unitID, -1)
+    local cmdList = {...} 
+    for i = 1, #commands do
+        for _, CMD in ipairs(cmdList) do
+            if commands[i].id == CMD or commands[i].id < 0 then                          
```

Comment:
`CMD` is a table so the numerical id will never be equal to it, should be `CMD.SOMETHING`